mingw-w64-git/PKGBUILD

```diff
@@ -50,7 +50,9 @@ build() {
   export PYTHON_PATH=/usr/bin/python2
   cd "$srcdir/git"
 
-  make -j1 all &&
```

Comment:
> There was no context for why this was here

You made me curious, so I looked at the [`blame` output](https://github.com/kevin-david/MINGW-packages/blame/6b3a57a7460186705da89dbdf3a22cdf9fcb9123/mingw-w64-git/PKGBUILD) and seeing as that is unhelpful, looking further with `git log -Sj1 -p mingw-w64-git/PKGBUILD`. It appears that indeed, it was added without explanation.

However, I have a good explanation for why I would like to keep the default at `-j1`: I personally build on a machine that also houses a continuous integration system, and frequently that means that I launch my builds and then go on to do other things. If all of those things started hogging all available cores, my machine would simply stop responding.

Plus, I did experience a couple of times that parallel builds hung without explanation; My hunch tells me that this might be a funny locking issue in the MSys runtime (which is hardly new, MSys1 has even worse locking issues).